realm/realm-library/src/main/cpp/io_realm_SyncSession.cpp

```diff
@@ -49,3 +50,64 @@ JNIEXPORT jboolean JNICALL Java_io_realm_SyncSession_nativeRefreshAccessToken(JN
     CATCH_STD()
     return JNI_FALSE;
 }
+
+
+JNIEXPORT jlong JNICALL Java_io_realm_SyncSession_nativeAddProgressListener(JNIEnv* env, jclass,
+                                                                            jstring localRealmPath, jlong listenerId,
```

Comment:
I thought we used `camelCase` for Java vars and `under_score` for c++ vars? I don't mind changing it, but having some way to tell the difference is kinda nice?